### PR TITLE
Add more checks to methods

### PR DESCRIPTION
**Pull request by MartinSchoeler**
@RocketChat/core 

Added more checks for methods in the packages/rocketchat-lib/server/methods/

### CODE REVIEW

File: server/methods/addRoomOwner.coffee [hunk 1]
```diff
@@ -1,11 +1,12 @@
 Meteor.methods
 	addRoomOwner: (rid, userId) ->
-		unless Meteor.userId()
-			throw new Meteor.Error 'error-invalid-user', 'Invalid user', { method: 'addRoomOwner' }
 
 		check rid, String
 		check userId, String
 
+		unless Meteor.userId()
+			throw new Meteor.Error 'error-invalid-user', 'Invalid user', { method: 'addRoomOwner' }
```
Comment (graywolf336) [inline server/methods/addRoomOwner.coffee]: Why was this moved down below the check?

File: packages/rocketchat-lib/server/methods/createChannel.coffee [hunk 2]
```diff
@@ -1,5 +1,9 @@
 Meteor.methods
 	createChannel: (name, members) ->
+
+		check name, String
+		check members, [String]
```
Comment (marceloschmidt) [inline packages/rocketchat-lib/server/methods/createChannel.coffee]: members is optional
If you check the function it calls `RocketChat.createRoom`, you'll see that members is not required, as it will be set to `[]` if none is passed.

File: packages/rocketchat-lib/server/methods/createPrivateGroup.coffee [hunk 3]
```diff
@@ -1,5 +1,9 @@
 Meteor.methods
 	createPrivateGroup: (name, members) ->
+
+		check name, String
+		check members, [String]
```
Comment (marceloschmidt) [inline packages/rocketchat-lib/server/methods/createPrivateGroup.coffee]: members is optional
If you check the function it calls RocketChat.createRoom, you'll see that members is not required, as it will be set to [] if none is passed.

File: server/methods/messageSearch.coffee [hunk 4]
```diff
@@ -1,5 +1,10 @@
 Meteor.methods
 	messageSearch: (text, rid, limit) ->
+
+		check text, String
+		check rid, String
+		check limit, Number
```
Comment (marceloschmidt) [inline server/methods/messageSearch.coffee]: limit is optional

File: server/methods/setAvatarFromService.coffee [hunk 5]
```diff
@@ -1,5 +1,10 @@
 Meteor.methods
 	setAvatarFromService: (dataURI, contentType, service) ->
+
+		check dataURI, String
+		check contentType, String
+		check service, String
```
Comment (marceloschmidt) [inline server/methods/setAvatarFromService.coffee]: contentType and service are optional

File: server/methods/setUserActiveStatus.coffee [hunk 6]
```diff
@@ -1,5 +1,8 @@
 Meteor.methods
 	setUserActiveStatus: (userId, active) ->
+
+		check userId, String
+
```
Comment (marceloschmidt) [inline server/methods/setUserActiveStatus.coffee]: I think it's ok to check if active is boolean

File: server/methods/addRoomOwner.coffee (hunk 1 above)
Comment (MartinSchoeler) [inline server/methods/addRoomOwner.coffee]: @graywolf336  I think its cheaper for the system to check first if the parameters are in their  correct type than to check if the user is valid, but i am not sure.

**Comment by marceloschmidt** [comment]
LGTM
